db/migrate/20160628140841_fix_service_order_placed_at.rb

```diff
@@ -0,0 +1,12 @@
+class FixServiceOrderPlacedAt < ActiveRecord::Migration[5.0]
+  class ServiceOrder < ActiveRecord::Base; end
+
+  def up
```

Comment:
@lpichler No need for a down migration as the `placed_at` field should have been populated all along.  This PR is back-filling that data.